common/containers/TabSection/Notifications.scss

```diff
@@ -142,3 +142,23 @@
     }
   }
 }
+
+.animation-enter {
+  opacity: 0.01;
+}
+
+.animation-enter.animation-enter-active {
+  opacity: 1;
+  transition: opacity 500ms;
+}
+
+.animation-exit {
+  opacity: 1;
+  transform: none;
+}
+
+.animation-exit.animation-exit-active {
+  opacity: 0.01;
+  transform: translateY(100%);
+  transition: opacity 500ms, transform 500ms;
+}
```

Comment:
Aaah, the ever familiar class naming of `react-transition-group`. Typically with these, I'm a big fan of:

```
.transitionname {
  &-enter {
    // ...
    &-active {
      // ...
    }
  }

  &-exit {
    // ...
    &-active {
      // ...
    }
  }
}
```

but this is completely stylistic

Reply:
I think your right here, it is stylistic but its good to maintain consistence. Still getting used to writing in suitcss. I'll update the css.

Reply:
Actually, looking at the style guide again, the css should be fine as is. I'm not styling children of the component so this wouldn't be correct
```
.transitionname {
  &-enter {
    // ...
    &-active {
      // ...
    }
  }
...
```
react-transition-group uses classes like `.transitionname-enter`& `.transitionname-enter-active`, to represent the [state of the component](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#componentnameis-stateofcomponent). React-transition-group doesn't adhere to the suitcss `.is-stateofcomponent` naming scheme, so sticking with the [react-transition-group naming scheme](https://reactcommunity.org/react-transition-group/#CSSTransition-prop-classNames) seems to be the only option 
```
.transitionname-enter {
     // ...
}

.transitionname-enter.transitionname-enter-active {
     // ...
}
```